src/python/bot/fuzzers/ml/gradientfuzz/model_utils.py

```diff
@@ -51,6 +51,9 @@ def update_state(self, *args, **kwargs):
       Computes (total matches) / (total branches).
       """
     y_true, y_pred = args
+    # TODO(mbarbella): Verify that this works properly and refactor this file
+    # for consistency between types if so.
```

Comment:
Maybe update the other update_state function in this file as well. can remove the todo as can refactor anyway if needed.

Reply:
Done!